### ISSUE TITLE
Issue in serializers perform_create

### ISSUE THREAD

**Issue by TralahM** [xtrinch/fcm-django#120]
```
File "/app/.heroku/python/lib/python3.6/site-packages/fcm_django/api/rest_framework.py", line 111, in perform_create
serializer.save(user=self.request.user, commit=False)
File "/app/.heroku/python/lib/python3.6/site-packages/rest_framework/serializers.py", line 187, in save
"'commit' is not a valid keyword argument to the 'save()' method. "
```
 **"'commit' is not a valid keyword argument to the 'save()' method. "**

 AssertionError: 'commit' is not a valid keyword argument to the 'save()' method. If you need to access data before committing to the database then inspect 'serializer.validated_data' instead. You can also pass additional keyword arguments to 'save()' if you need to set extra attributes on the saved model instance. For example: 'serializer.save(owner=request.user)'.'

**Comment by xtrinch** [comment]
Update the package and try again

**Comment by TralahM** [comment]
okay resolved installed version 0.3.2  :heavy_check_mark:

**Comment by xtrinch** [comment]
I was talking more about 0.3.4, but OK if that works for you
